static/js/components/StarList.jsx

```diff
@@ -1,11 +1,29 @@
 import React, { useEffect, useState } from 'react';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 
 import { GET } from '../API';
 
 import styles from './StarList.css';
 
+function starListElem(starList) {
```

Comment:
Again, prefer these to be React components

Reply:
Also, does `Elem` mean "element"? If so, it looks like the name is misleading, as it doesn't just render a single element of the starlist, but rather every element in the starlist. Or the name could just be more explicit/clearer

Reply:
Made this a react component called `StarListBody`.